<?xml version="1.0"?>
<launch>
    <arg name="hostname" default="192.168.0.1"/>
    <arg name="frame_id" default="cloud"/>
    <arg name="sw_pll_only_publish" default="True"/>
    <node name="sick_lrs_36xx" pkg="sick_scan" type="sick_generic_caller" respawn="false" output="screen">
        <param name="intensity" type="bool" value="True"/>
        <param name="frame_id" type="str" value="$(arg frame_id)"/>
        <param name="use_binary_protocol" type="bool" value="true"/>
        <param name="scanner_type" type="string" value="sick_lrs_36xx"/>
        <param name="range_max" type="double" value="25.0"/>
        <param name="hostname" type="string" value="$(arg hostname)"/>
        <param name="port" type="string" value="2112"/>
        <param name="timelimit" type="int" value="5"/>
        <param name="sw_pll_only_publish" type="bool" value="$(arg sw_pll_only_publish)"/>
        <param name="start_services" type="bool" value="True"/> <!-- start ros service for cola commands -->
        <param name="activate_lferec" type="bool" value="True"/> <!-- activate field monitoring by lferec messages -->
        <param name="activate_lidinputstate" type="bool" value="False"/>
        <param name="activate_lidoutputstate" type="bool" value="False"/>
        <param name="scan_cfg_list_entry" type="int" value="6"/> <!-- |5  |0x |6 Hz   |6 Hz    |0.1875째 |0.1875째  |360째   |0 ...  360째      |x  |x  |x  |(x)|(x)|-->
    </node>
</launch>
